obspy/io/nordic/ellipse.py

```diff
@@ -12,9 +12,6 @@
 - See if a point is inside or on an ellipse
 - Calculate the angle subtended by an ellipse (for back-azimuth uncertainty)
 - Plot an ellipse
-
-.. note:
- TODO: ellipsoids (3D ellipses)
```

Comment:
Do we now have full 3D ellipsoids?

Reply:
The code is written (I think I sent it to you), but would be best put under a more general path than io.nordic (core.util?) to take advantage of some of the things that can be done with Ellipsoids (back-azimuth uncertainty calculations, plots with ellipsoids...).  I don't have the resources to do this anytime soon, but could help someone else who was willing to.

Reply:
Okay, I am keen to have a go some time, but I don't have time either at the moment, maybe once term has finished... Thanks, and yes you did send it to me.